docs/components/base/sensor-controlled.md

```diff
@@ -14,9 +14,11 @@ aliases:
 A `sensor-controlled` base supports a robotic base with feedback control from a movement sensor.
 
 {{% alert title="Requirements" color="note" %}}
-In order to use feedback control for [SetVelocity()](/components/base/#setvelocity), you must provide a movement sensor that implements [AngularVelocity()](/components/movement-sensor/#getangularvelocity) and [LinearVelocity()](/components/movement-sensor/#getlinearvelocity).
+In order to use feedback control, you must provide a movement sensor that implements [AngularVelocity()](/components/movement-sensor/#getangularvelocity) and [LinearVelocity()](/components/movement-sensor/#getlinearvelocity). This will enable feedback control for [SetVelocity()](/components/base/#setvelocity).
```

Comment:
the distinction I wanted to add here is that all 3 methods require `AngularVelocity()` and `LinearVelocity()`. Not sure if theres a better way to say this